# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musicbrainz-trm/gcc43.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- trm-0.2.1/mbid3.cpp.orig	2010-08-03 12:37:48.596086856 +0200
+++ trm-0.2.1/mbid3.cpp	2010-08-03 12:38:32.488255868 +0200
@@ -6,6 +6,7 @@
 #ifdef HAVE_LIBID3
 
 #include <stdio.h>
+#include <string.h>
 #include "mbid3.h"
 
 #define DB Debug_v("%s:%d", __FILE__, __LINE__);
